daml-assistant/integration-tests/BUILD.bazel

```diff
@@ -71,14 +71,19 @@ da_haskell_test(
     name = "integration-tests",
     timeout = "long",
     srcs = glob(["src/**/*.hs"]),
+    args = [
+        "$(location //:yarn)",
+        "daml-types-not-available" if is_windows else "$(location //language-support/ts/daml-types:npm_package)",
```

Comment:
We should leave a comment here that this test is not run in Windows but we need to pass something to make the test harness happy.

Reply:
Ok